src/components/include/utils/threads/thread.h

```diff
@@ -90,7 +98,12 @@ void sleep(uint32_t ms);
 Thread* CreateThread(const char* name, ThreadDelegate* delegate);
 void DeleteThread(Thread* thread);
 
+#ifdef QT_PORT
+class Thread : public QObject {
```

Comment:
that were available QT signals and slots mechanism.
QT MOC appended signals & slots for this class